test-framework/common/src/main/java/io/quarkus/test/common/NativeImageLauncher.java

```diff
@@ -109,11 +109,24 @@ private static String guessPath(Class<?> testClass) {
                             return file.getAbsolutePath();
                         }
                     }
+                } else if (url.getProtocol().equals("file") && url.getPath().contains("/target/surefire/")) {
```

Comment:
Will these paths work on Windows?

Reply:
I am not sure, it is a URL path so I think so, either way windows support for native is pretty limited at the moment AFAIK, and the runner would probably have a .exe extension so that would not work either.

Reply:
Makes sense